DeviceAdapters/DemoCamera/DemoCamera.h

```diff
@@ -174,7 +175,9 @@ class CDemoCamera : public CCameraBase<CDemoCamera>
    // action interface
    // ----------------
    int OnMaxExposure(MM::PropertyBase* pProp, MM::ActionType eAct);
-	int OnTestProperty(MM::PropertyBase* pProp, MM::ActionType eAct, long);
+   int OnTestProperty(MM::PropertyBase* pProp, MM::ActionType eAct, long);
```

Comment:
This is just fixing the indenting from tab to spaces in order to match the rest of the file.